### PR TITLE
Fix mpris dbus notifications

### PR DESCRIPTION
**Pull request by printesoi**
Clementine does not send the correct signal about the changed properties such as CanGoNext, CanGoPrevious, CanSeek and may cause programs that listen to those changes use the wrong information. 

To check the corect signals are send one can use:

`dbus-monitor --session --monitor "type='signal',interface=org.freedesktop.DBus.Properties,member=PropertiesChanged,path=/org/mpris/MediaPlayer2"`

### CODE REVIEW

File: src/core/mpris2.cpp [hunk 1]
```diff
@@ -404,6 +421,14 @@ bool Mpris2::CanSeek() const {
   }
 }
 
+bool Mpris2::CanSeek(Engine::State state) const {
+    if (mpris1_->player()) {
```
Comment (hatstand) [inline src/core/mpris2.cpp]: Use a ternary expression

**Comment by hatstand** [comment]
Can you also run `make format`?

**Comment by printesoi** [comment]
@hatstand I did run `make format`, I get:

    src/core/mpris2.cpp: formatting is correct!
    src/core/mpris2.h: formatting is correct!
    src/playlist/playlist.cpp: formatting is correct!

**Comment by printesoi** [comment]
@hatstand I ran `make format`

File: src/core/mpris2.cpp (hunk 1 above)
Comment (printesoi) [inline src/core/mpris2.cpp]: Fixed.

**Pull request merged**